userdata/xDynamicConfig.ps1

```diff
@@ -150,7 +150,8 @@ Configuration xDynamicConfig {
     'RegistryKeySet' = 'Registry';
     'RegistryValueSet' = 'Registry';
     'DisableIndexing' = 'Script';
-    'FirewallRule' = 'Script'
+    'FirewallRule' = 'Script';
```

Comment:
i'm puzzled why git diff sees any change here. i believe the changes to add ReplaceInFile to xDynamicConfig.ps1 is already in master.
eg:
- https://github.com/mozilla-releng/OpenCloudConfig/blob/master/userdata/xDynamicConfig.ps1#L154
- https://github.com/mozilla-releng/OpenCloudConfig/blob/master/userdata/xDynamicConfig.ps1#L660-L674